 AH00491: caught SIGTERM, shutting down
 AH00489: Apache/2.4.29 (Ubuntu) mod_wsgi/4.5.17 Python/3.6 configured -- resuming normal operations
 AH00094: Command line: '/usr/sbin/apache2'

 Traceback (most recent call last):
   File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 2292, in wsgi_app
     response = self.full_dispatch_request()
   File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1815, in full_dispatch_request
     rv = self.handle_user_exception(e)
   File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1718, in handle_user_exception
     reraise(exc_type, exc_value, tb)
   File "/usr/local/lib/python3.6/dist-packages/flask/_compat.py", line 35, in reraise
     raise value
   File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1813, in full_dispatch_request
     rv = self.dispatch_request()
   File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1799, in dispatch_request
(**req.view_args)
   File "/var/www/BigToe/BigToe/__init__.py", line 27, in landing
     return render_template(str(os.path))
   File "/usr/local/lib/python3.6/dist-packages/flask/templating.py", line 134, in render_template
     return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
   File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 869, in get_or_select_template
     return self.get_template(template_name_or_list, parent, globals)
   File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 830, in get_template
     return self._load_template(name, self.make_globals(globals))
   File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 804, in _load_template
     template = self.loader.load(self, name, globals)
   File "/usr/lib/python3/dist-packages/jinja2/loaders.py", line 113, in load
     source, filename, uptodate = self.get_source(environment, name)
   File "/usr/local/lib/python3.6/dist-packages/flask/templating.py", line 58, in get_source
     return self._get_source_fast(environment, template)
   File "/usr/local/lib/python3.6/dist-packages/flask/templating.py", line 86, in _get_source_fast
     raise TemplateNotFound(template)
 jinja2.exceptions.TemplateNotFound: <module 'posixpath' from '/usr/lib/python3.6/posixpath.py'>
 
